src/vs/workbench/contrib/terminal/browser/media/shellIntegration-rc.zsh

```diff
@@ -112,8 +112,8 @@ __vsc_update_cwd() {
 }
```

Comment:
why were these moved to below?

Reply:
To get the command line before command execute fires, this should greatly improve the results and likely other features from shell integration too